sickbeard/providers/kat.py

```diff
@@ -166,16 +126,17 @@ def _doSearch(self, search_strings, search_mode='eponly', epcount=0, age=0, epOb
                         except (AttributeError, TypeError, KeyError):
                             continue
 
-                        # Dont let RSS add items with no seeders either -.-
-                        if not seeders or seeders < self.minseed or leechers < self.minleech:
-                            logger.log(u"Discarding torrent because it doesn't meet the minimum seeders or leechers: {0} (S:{1} L:{2})".format(title, seeders, leechers), logger.DEBUG)
+                        if not all([title, download_url]):
                             continue
 
-                        if self.confirmed and not verified:
-                            logger.log(u"KAT Provider found result " + title + " but that doesn't seem like a verified result so I'm ignoring it", logger.DEBUG)
+                        #Filter unseeded torrent
+                        if seeders < self.minseed or leechers < self.minleech:
+                            if mode != 'RSS':
+                                logger.log(u"Discarding torrent because it doesn't meet the minimum seeders or leechers: {0} (S:{1} L:{2})".format(title, seeders, leechers), logger.DEBUG)
                             continue
 
-                        if not title or not url:
+                        if self.confirmed and not verified and mode != RSS:
+                            logger.log(u"Found result " + title + " but that doesn't seem like a verified result so I'm ignoring it", logger.DEBUG)
```

Comment:
@miigotu getting this error.

Strange as mode is used a few lines above. Do you know what's wrong?

```
  File "/home/osmc/SickRage/sickbeard/providers/kat.py", line 139, in _doSearch
    if self.confirmed and not verified and mode != RSS:
NameError: global name 'RSS' is not defined

```